lib/node_modules/@stdlib/math/base/special/log2/examples/c/example.c

```diff
@@ -0,0 +1,33 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/log2.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+int main( void ) {
+    double x;
+    double v;
+    int i;
+    
+    for ( i = 0; i < 100; i++ ) {
+        x = ( ( (double)rand() / (double)RAND_MAX ) * 100.0 );
+        v = stdlib_base_log2( x );
+        printf( "log2(%lf) = %lf\n", x, v );
+    }
```

Comment:
This should be TAB indentation. Again, please make sure you've setup EditorConfig, as reference in the development guide: https://github.com/stdlib-js/stdlib/blob/58c9a5053430b55c974e9ecccfc45c51c3fefc9e/.editorconfig#L71.

Reply:
Thanks, I might have missed that, I have setup EditorConfig.